modules/cpmso-yaml-failure-domain-aws.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
+//
+// * machine_management/control-plane-machine-management.adoc
+
+:_content-type: REFERENCE
+[id="cpmso-yaml-failure-domain-aws_{context}"]
+= Sample AWS failure domain configuration
+
+The `ControlPlaneMachineSet` CR spreads control plane machines across multiple failure domains when possible.
+
+.Sample AWS failure domain values
+[source,yaml]
+----
+failureDomains:
+  aws:
+  - placement:
+      availabilityZone: <aws_region_1> <a>
+    subnet:
+      filters:
+      - name: tag:Name <b>
+        values:
+        - <cluster_id>-private-<aws_region_1> <c>
+      type: Filters
```

Comment:
There are other types, you can also use ID or ARN, which may be helpful for people to know?